src/reaper_osara.cpp

```diff
@@ -3756,6 +3778,53 @@ void cmdMoveToPrevItem(Command* command) {
 	}
 }
 
+void reportNudgeAndZoomStep(double nudgeTime) {
+	TimeFormat tf = TF_MILSEC;
+	if (nudgeTime >= 1.0) tf = TF_ROUNDSEC;
+	outputMessage(formatTime(nudgeTime, tf));
+}
+
+void nudgeAndZoomStep(int direction) {
+	// direction = +1 → zoom out (increase nudge time), -1 → zoom in (decrease nudge time)
+	double currentPPS = GetHZoomLevel();
+	if (currentPPS <= 0.0) {
+		outputMessage(translate("invalid zoom level"));
+		return;
+	}
+	double currentNudgeTime = 1.0 / currentPPS;
+	double nextNudgeTime = currentNudgeTime;
+	if (direction > 0) {
+		// Find next larger nudge
+		for (int i = 0; i < NUDGE_COUNT; ++i) {
```

Comment:
This can be simplified using a range-based for loop:
`for (double value : NUDGE_VALUES) {`
This also means we can get rid of NUDGE_COUNT.